linode/manager
# upcoming: [M3-7302] - Replace NodeBalancer detail charts with Recharts

## Description 📝
This PR is part of a bigger effort to replace chart.js with the more modern Recharts. 

## Changes  🔄
List any change relevant to the reviewer.
- Replace the NodeBalancer details page charts with Recharts
- Refactor code to use one AreaChart component

## Preview 📷
| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/linode/manager/assets/115299789/29ff3568-bb89-4bff-aa5f-55d027355ae0" /> | <video src="https://github.com/linode/manager/assets/115299789/58d1e6a2-07d3-4cf6-8b9b-c3c6cc209c62" /> |
| <video src="https://github.com/linode/manager/assets/115299789/13afd4c3-389b-4bba-a4cf-908fbc246cf8" /> | <video src="https://github.com/linode/manager/assets/115299789/c32de831-c5fc-445e-957f-294840aa0052" /> |

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Have a NodeBalancer that's been running for a while

### Verification steps 
(How to verify changes)
- Go to the Network tab of a NodeBalancer that's been running for a while
  - Confirm the updated UI and that there are no regressions in the graph, units, etc
- Confirm there are no regressions in the Linode Network Transfer History chart from the refactoring

## As an Author I have considered 🤔

*Check all that apply*
